sql/core/benchmarks/DataSourceReadBenchmark-jdk21-results.txt

```diff
@@ -2,430 +2,430 @@
+SQL Parquet MR: DataPageV2                                            2387           2398          16          6.6         151.8       1.0X
+SQL Parquet Vectorized: DataPageV2 (Nested Column Disabled)           2865           2869           6          5.5         182.1       0.8X
+SQL Parquet Vectorized: DataPageV2 (Nested Column Enabled)             276            282           4         57.0          17.5       8.6X
+
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 SQL Single BIGINT Column Scan in Struct:                     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------------------------
-SQL ORC MR                                                            2253           2258           7          7.0         143.3       1.0X
-SQL ORC Vectorized (Nested Column Disabled)                           2311           2324          18          6.8         146.9       1.0X
-SQL ORC Vectorized (Nested Column Enabled)                             356            377          29         44.2          22.6       6.3X
-SQL Parquet MR: DataPageV1                                            2600           2609          13          6.0         165.3       0.9X
-SQL Parquet Vectorized: DataPageV1 (Nested Column Disabled)           3090           3097           9          5.1         196.5       0.7X
-SQL Parquet Vectorized: DataPageV1 (Nested Column Enabled)             128            144          16        122.6           8.2      17.6X
-SQL Parquet MR: DataPageV2                                            2303           2325          31          6.8         146.4       1.0X
-SQL Parquet Vectorized: DataPageV2 (Nested Column Disabled)           2816           2821           7          5.6         179.0       0.8X
-SQL Parquet Vectorized: DataPageV2 (Nested Column Enabled)             332            339           7         47.3          21.1       6.8X
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
+SQL ORC MR                                                            2207           2213           9          7.1         140.3       1.0X
+SQL ORC Vectorized (Nested Column Disabled)                           2245           2267          32          7.0         142.7       1.0X
+SQL ORC Vectorized (Nested Column Enabled)                             285            305          38         55.1          18.1       7.7X
+SQL Parquet MR: DataPageV1                                            2807           2812           8          5.6         178.4       0.8X
+SQL Parquet Vectorized: DataPageV1 (Nested Column Disabled)           3405           3407           3          4.6         216.5       0.6X
+SQL Parquet Vectorized: DataPageV1 (Nested Column Enabled)             291            321          20         54.0          18.5       7.6X
```

Comment:
This also shows a regression on `DataPageV1` (according to the ratio change of `DataPageV1` vs `DataPageV2`. `DataPageV1` becomes slower than `DataPageV2`.